src/shared/components/case-editor/services/cases.service.ts

```diff
@@ -9,6 +9,7 @@ import { catchError, map, tap } from 'rxjs/operators';
 import { CaseEventData, CaseEventTrigger, CasePrintDocument, CaseView, Draft } from '../../../domain';
 import { WizardPage } from '../domain';
 import { WizardPageFieldToCaseFieldMapper } from './wizard-page-field-to-case-field.mapper';
+import { _ } from 'underscore';
```

Comment:
Why are we importing this?

Reply:
I guess this was already removed in the tsLint changes , may be it did not reflect it